lib/diagnostic.js

```diff
@@ -38,19 +54,36 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Bubbles';
+  species: 'Dog';
+  breed: 'Lab';
+  vocalization: function() {
+    this.vocalize
+  }
+  return vocalize;
+}
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
-};
+const Pet = function(name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
+  };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function() {
+  vocalization: function() {
+    this.vocalize
+  }
+  return vocalize;
+}
+}
```

Comment:
You correctly added a method to the prototype (`Pet.prototype.methodName = function(){}`), but the function you added is wrong. See my notes on Q3, above.